test/bats/tests/azure_v1alpha1_secretproviderclass.yaml

```diff
@@ -1,42 +0,0 @@
-apiVersion: secrets-store.csi.x-k8s.io/v1alpha1
```

Comment:
Can you pls confirm we have an example of each of these spc test scenarios in the /examples/ folder?

Reply:
Refactored the `examples/` folder to add more specific examples for configurations we support.